api/src/main/java/org/pmiops/workbench/config/WorkbenchConfigMapper.java

```diff
@@ -53,6 +53,15 @@ public interface WorkbenchConfigMapper {
   @Mapping(
       target = "enableInitialCreditsExpiration",
       source = "config.featureFlags.enableInitialCreditsExpiration")
+  @Mapping(
+      target = "initialCreditsValidityPeriodDays",
+      source = "config.billing.initialCreditsValidityPeriodDays")
+  @Mapping(
```

Comment:
This represents how many days will be added to the credit start date in order to calculate the date that initial credits will expire after an extension. This is not currently used in the UI, because we are not mentioning this date until the extension as already happened. Happy to remove this if we do not end up needing it.